calculator/operations.py

```diff
@@ -1,18 +1,29 @@
+from sympy.plotting import plot
 
 def add(*args):
     # your implementation here
-    pass
+    return operate(lambda x, y : x + y, *args)
 
 def subtract(*args):
     # your implementation here
-    pass
+    return operate(lambda x, y: x - y, *args)
 
 def multiply(*args):
     # your implementation here
-    pass
+    return operate(lambda x, y : x * y, *args)
 
 def divide(*args):
     # your implementation here
-    pass
+    return operate(lambda x, y : float(x) / y, *args)
 
 # add your custom operations here
+def operate(operation, *args):
+    if not args:
+        return 0
+    
+    output = args[0]
+    for number in args[1:]:
+        output = operation(output, number)
+    return output
```

Comment:
I like the lambda to operate function. A question -- If there was only one item in args, would args[1:] throw an exception?